<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::ClickFocus</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>311</width>
     <height>305</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="settings_page_layout">
    <item>
     <widget class="QLabel" name="resumeLabel">
      <property name="text">
       <string>If you are in game press settings key to resume</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="setttings_page_sublayout">
      <item>
       <layout class="QHBoxLayout" name="settingsLayout">
        <item>
         <layout class="QVBoxLayout" name="settingsLabelLayout">
          <item>
           <widget class="QLabel" name="volume">
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="settings">
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pause">
            <property name="text">
             <string>Pause game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mvUp">
            <property name="text">
             <string>Move up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mvLeft">
            <property name="text">
             <string>Move left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mvDown">
            <property name="text">
             <string>Move down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mvRight">
            <property name="text">
             <string>Move right</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="settingsButtonsLayout">
          <item>
           <widget class="QSlider" name="volumeSlider">
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="settingsBinder">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseBinder">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mvUpBinder">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mvLeftBinder">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mvDownBinder">
            <property name="cursor">
             <cursorShape>OpenHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mvRightBinder">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string/>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Back to main menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
